OpenRA.Game/Graphics/Viewport.cs

```diff
@@ -62,7 +62,7 @@ public float Zoom
 
 			set
 			{
-				zoom = value;
+				zoom = value.Clamp(0.25f, 2f);
```

Comment:
I think it is a bad idea to arbitrarily restrict the zoom here.  If you want to limit the amount that the user can zoom in the UI, then this should be done in the UI code.